Sources/Plasma/NucleusLib/pnInputCore/plInputMap.cpp

```diff
@@ -750,6 +754,65 @@ const std::map<uint32_t, ST::string> plKeyMap::fKeyConversionEnglish =
     { VK_OEM_5,     ST_LITERAL("Backslash") },
     { VK_OEM_6,     ST_LITERAL("RightBracket") },
     { VK_OEM_7,     ST_LITERAL("Quote") },
+#elif HS_BUILD_FOR_APPLE
+    { kVK_F1,    "F1"},
```

Comment:
Would be good to wrap the key name strings in `ST_LITERAL()` as was done for the Windows keys

Reply:
Actually, the lookup here happens by Plasma `KEY_` defines (which are mapped to platform-specific virtual keys), so if we change everything from `VK_` to `KEY_` we shouldn't need platform-specific mappings here.

I did this in my Linux client prep branch, but would be easy to cherry-pick just the one file over: https://github.com/H-uru/Plasma/pull/1325/commits/ed4639950e246259eb38079bff6237f28232d40e#diff-97f3aa4f309c9c37f5730e71fe97878bfbc87eb021259a5a733973af7e2d04d4

Reply:
Sorry, took me a little bit to backtrack through the code and figure out what this comment meant. I can make the changeover. All the other localizations should eventually make the changeover - but right now English is the only one that was touched for Mac.